applications/crossbar/src/modules_v1/cb_phone_numbers_v1.erl

```diff
@@ -461,6 +462,8 @@ validate_delete(Context) ->
                           cb_context:context().
 set_response({'ok', {'ok', Doc}}, _, Context) ->
     crossbar_util:response(Doc, Context);
+set_response({'ok', #knm_number{}=Number}, _, Context) ->
```

Comment:
At first, I created this function, but then changed my mind and decided to use record.
Ok. I will redo changes  and add function.